spec/models/order_spec.rb

```diff
@@ -21,6 +21,22 @@
     end
   end
 
+  describe "#save_card" do
+    it "creates a Stripe customer" do
+      order = Order.create(user: user, stripe_token: stripe_helper.generate_card_token)
+      credit_card = CreditCard.create(user: user)
+      order.save_card
+      expect(Stripe::Customer.all).to_not be_empty
+    end
+    it "saves a credit card on a user's first order" do
+      order = Order.create(user: user, stripe_token: stripe_helper.generate_card_token)
+      credit_card = CreditCard.create(user: user)
+      order.save_card
+      expect(order.credit_card.card_token).to_not eq nil
+      expect(order.credit_card.last_four_digits).to_not eq nil
```

Comment:
If YOU think it's more readable, change `eq nil` to `be_nil`. I personally do but it's your call!